lib/Lolifiers/NyanCat.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Monolol\Lolifiers;
+
+use Monolol\Lolifier;
+
+class NyanCat implements Lolifier
+{
+    public function isHandling(array $record)
+    {
+        return true;
+    }
+
+    public function lolify(array $record)
+    {
+        $message['message'] = preg_replace("/[A-Za-zÀ-ÿ0-9]+/", "nya", $record['message']);
```

Comment:
Be careful,you have to return ```$record```. Message is replaced but other keys (like level) must be kept.